README.md

```diff
@@ -81,6 +81,23 @@ read(url, {
   });
 ```
 
+- `candidatesFilters` which allow set your own filters for candidate tags.
+
+options.candidatesFilters = [callback(obj, index)]
+```javascript
+read(url, {
+  candidatesFilters: [
+    function (obj) {
+      if (obj.tagName === 'ARTICLE' && elem.getAttribute('type') === 'video') {
```

Comment:
Also think `elem` is not defined in this example. I think you meant to make it the same as `obj`.